bin/citrea/src/rollup/mod.rs

```diff
@@ -344,4 +345,100 @@ pub trait CitreaRollupBlueprint: RollupBlueprint {
             rpc_methods,
         })
     }
+
+    /// Creates a new light client prover
+    #[instrument(level = "trace", skip_all)]
+    async fn create_new_light_client_prover(
+        &self,
+        runtime_genesis_paths: &<Self::NativeRuntime as RuntimeTrait<
+            Self::NativeContext,
+            Self::DaSpec,
+        >>::GenesisPaths,
+        rollup_config: FullNodeConfig<Self::DaConfig>,
+        prover_config: LightClientProverConfig,
+    ) -> Result<LightClientProver<Self>, anyhow::Error>
+    where
+        <Self::NativeContext as Spec>::Storage: NativeStorage,
+    {
+        let da_service = self.create_da_service(&rollup_config, true).await?;
+
+        let rocksdb_config = RocksdbConfig::new(
+            rollup_config.storage.path.as_path(),
+            rollup_config.storage.db_max_open_files,
+        );
+        let ledger_db = self.create_ledger_db(&rocksdb_config);
+
+        let prover_service = self
+            .create_batch_prover_service(
```

Comment:
you need to implement create_light_client_prover_service, since that's where we pass the zkvm ids